app/controllers/checkouts_controller.rb

```diff
@@ -181,5 +179,11 @@ def checkin_bak
       end
     end
   end
+
+  # def reply
```

Comment:
I hear what you're saying, but checking in commented out code is always bad practice. If you really want to save this for next year I suggest creating a wiki page or some other documentation about what your aims were including any un-finished code.